mne/viz/_3d.py

```diff
@@ -683,11 +683,13 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
             if head_surf is None:
                 subject_dir = op.join(
                     get_subjects_dir(subjects_dir, raise_error=True), subject)
-                if s in ('head-dense', 'seghead'):
+                if 'head' in s:  # 'head-dense', 'seghead', 'head'
```

Comment:
I don't think we want to change the logic here, but rather add `-head.fif` to this list and the one a few lines lower with `outer_skin.surf`s

Reply:
... actually `-head.fif` should probably *only* be added to the list below, since `-head-dense.fif` is already in this list

Reply:
yup, makes sense